From 48cf1b4997b3da46b0a726a5f0d903c2559689e8 Mon Sep 17 00:00:00 2001
From: "K.Takata" <kentkt@csc.jp>
Date: Tue, 23 Oct 2018 01:25:54 +0900
Subject: [PATCH 30/30] nsis: Adjust some messages and adjust layout for them

* Clarify that loading defaults.vim or vimrc_example.vim.
  (Based on the German translation.)
* Clarify that remap some keys for Windows.
---
 nsis/gvim.nsi             | 16 ++++++++--------
 nsis/lang/dutch.nsi       |  6 +++---
 nsis/lang/english.nsi     |  6 +++---
 nsis/lang/italian.nsi     |  6 +++---
 nsis/lang/japanese.nsi    |  6 +++---
 nsis/lang/simpchinese.nsi |  6 +++---
 nsis/lang/tradchinese.nsi |  6 +++---
 7 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/nsis/gvim.nsi b/nsis/gvim.nsi
index d57de8d2d..1b687d236 100644
--- a/nsis/gvim.nsi
+++ b/nsis/gvim.nsi
@@ -725,12 +725,12 @@ Function SetCustom
 
 
 	# 1st group - Compatibility
-	${NSD_CreateGroupBox} 0 0 100% 24% $(str_msg_compat_title)
+	${NSD_CreateGroupBox} 0 0 100% 32% $(str_msg_compat_title)
 	Pop $3
 
 	${NSD_CreateLabel} 5% 10% 35% 8% $(str_msg_compat_desc)
 	Pop $3
-	${NSD_CreateDropList} 48% 10% 45% 8% ""
+	${NSD_CreateDropList} 18% 19% 75% 8% ""
 	Pop $vim_nsd_compat
 	${NSD_CB_AddString} $vim_nsd_compat $(str_msg_compat_vi)
 	${NSD_CB_AddString} $vim_nsd_compat $(str_msg_compat_vim)
@@ -756,12 +756,12 @@ Function SetCustom
 
 
 	# 2nd group - Key remapping
-	${NSD_CreateGroupBox} 0 27% 100% 33% $(str_msg_keymap_title)
+	${NSD_CreateGroupBox} 0 35% 100% 31% $(str_msg_keymap_title)
 	Pop $3
 
-	${NSD_CreateLabel} 5% 37% 90% 8% $(str_msg_keymap_desc)
+	${NSD_CreateLabel} 5% 45% 90% 8% $(str_msg_keymap_desc)
 	Pop $3
-	${NSD_CreateDropList} 48% 47% 45% 8% ""
+	${NSD_CreateDropList} 38% 54% 55% 8% ""
 	Pop $vim_nsd_keymap
 	${NSD_CB_AddString} $vim_nsd_keymap $(str_msg_keymap_default)
 	${NSD_CB_AddString} $vim_nsd_keymap $(str_msg_keymap_windows)
@@ -781,12 +781,12 @@ Function SetCustom
 
 
 	# 3rd group - Mouse behavior
-	${NSD_CreateGroupBox} 0 63% 100% 36% $(str_msg_mouse_title)
+	${NSD_CreateGroupBox} 0 69% 100% 31% $(str_msg_mouse_title)
 	Pop $3
 
-	${NSD_CreateLabel} 5% 73% 90% 8% $(str_msg_mouse_desc)
+	${NSD_CreateLabel} 5% 79% 90% 8% $(str_msg_mouse_desc)
 	Pop $3
-	${NSD_CreateDropList} 23% 82% 70% 8% ""
+	${NSD_CreateDropList} 23% 87% 70% 8% ""
 	Pop $vim_nsd_mouse
 	${NSD_CB_AddString} $vim_nsd_mouse $(str_msg_mouse_default)
 	${NSD_CB_AddString} $vim_nsd_mouse $(str_msg_mouse_windows)
diff --git a/nsis/lang/dutch.nsi b/nsis/lang/dutch.nsi
index 5e9f8e61a..f37387eb4 100644
--- a/nsis/lang/dutch.nsi
+++ b/nsis/lang/dutch.nsi
@@ -249,14 +249,14 @@ LangString str_msg_compat_vi       ${LANG_DUTCH} \
 LangString str_msg_compat_vim      ${LANG_DUTCH} \
     "Vim original"
 LangString str_msg_compat_defaults ${LANG_DUTCH} \
-    "Vim with some enhancements"
+    "Vim with some enhancements (load defaults.vim)"
 LangString str_msg_compat_all      ${LANG_DUTCH} \
-    "Vim with all enhancements (Default)"
+    "Vim with all enhancements (load vimrc_example.vim) (Default)"
 
 LangString str_msg_keymap_title   ${LANG_DUTCH} \
     " Mappings "
 LangString str_msg_keymap_desc    ${LANG_DUTCH} \
-    "&Remap a few keys (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
 LangString str_msg_keymap_default ${LANG_DUTCH} \
     "Do not remap keys (Default)"
 LangString str_msg_keymap_windows ${LANG_DUTCH} \
diff --git a/nsis/lang/english.nsi b/nsis/lang/english.nsi
index fd7ed5ec1..88fed67a4 100644
--- a/nsis/lang/english.nsi
+++ b/nsis/lang/english.nsi
@@ -258,14 +258,14 @@ LangString str_msg_compat_vi       ${LANG_ENGLISH} \
 LangString str_msg_compat_vim      ${LANG_ENGLISH} \
     "Vim original"
 LangString str_msg_compat_defaults ${LANG_ENGLISH} \
-    "Vim with some enhancements"
+    "Vim with some enhancements (load defaults.vim)"
 LangString str_msg_compat_all      ${LANG_ENGLISH} \
-    "Vim with all enhancements (Default)"
+    "Vim with all enhancements (load vimrc_example.vim) (Default)"
 
 LangString str_msg_keymap_title   ${LANG_ENGLISH} \
     " Mappings "
 LangString str_msg_keymap_desc    ${LANG_ENGLISH} \
-    "&Remap a few keys (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
 LangString str_msg_keymap_default ${LANG_ENGLISH} \
     "Do not remap keys (Default)"
 LangString str_msg_keymap_windows ${LANG_ENGLISH} \
diff --git a/nsis/lang/italian.nsi b/nsis/lang/italian.nsi
index 53c6980f9..28dfb280b 100644
--- a/nsis/lang/italian.nsi
+++ b/nsis/lang/italian.nsi
@@ -258,14 +258,14 @@ LangString str_msg_compat_vi       ${LANG_ITALIAN} \
 LangString str_msg_compat_vim      ${LANG_ITALIAN} \
     "Vim original"
 LangString str_msg_compat_defaults ${LANG_ITALIAN} \
-    "Vim with some enhancements"
+    "Vim with some enhancements (load defaults.vim)"
 LangString str_msg_compat_all      ${LANG_ITALIAN} \
-    "Vim with all enhancements (Default)"
+    "Vim with all enhancements (load vimrc_example.vim) (Default)"
 
 LangString str_msg_keymap_title   ${LANG_ITALIAN} \
     " Mappings "
 LangString str_msg_keymap_desc    ${LANG_ITALIAN} \
-    "&Remap a few keys (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
 LangString str_msg_keymap_default ${LANG_ITALIAN} \
     "Do not remap keys (Default)"
 LangString str_msg_keymap_windows ${LANG_ITALIAN} \
diff --git a/nsis/lang/japanese.nsi b/nsis/lang/japanese.nsi
index ad86242c3..ce3725443 100644
--- a/nsis/lang/japanese.nsi
+++ b/nsis/lang/japanese.nsi
@@ -254,14 +254,14 @@ LangString str_msg_compat_vi       ${LANG_JAPANESE} \
 LangString str_msg_compat_vim      ${LANG_JAPANESE} \
     "Vim 独自"
 LangString str_msg_compat_defaults ${LANG_JAPANESE} \
-    "Vim 独自と多少の拡張"
+    "Vim 独自と多少の拡張 (defaults.vim を読み込み)"
 LangString str_msg_compat_all      ${LANG_JAPANESE} \
-    "Vim 独自と全ての拡張 (既定)"
+    "Vim 独自と全ての拡張 (vimrc_example.vim を読み込み) (既定)"
 
 LangString str_msg_keymap_title   ${LANG_JAPANESE} \
     " マッピング "
 LangString str_msg_keymap_desc    ${LANG_JAPANESE} \
-    "いくつかのキーをリマップする(&R) (例: Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F など)"
+    "Windows用に一部のキーをリマップする(&R) (例: Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F など)"
 LangString str_msg_keymap_default ${LANG_JAPANESE} \
     "リマップしない (既定)"
 LangString str_msg_keymap_windows ${LANG_JAPANESE} \
diff --git a/nsis/lang/simpchinese.nsi b/nsis/lang/simpchinese.nsi
index 623579a5e..2f6378603 100644
--- a/nsis/lang/simpchinese.nsi
+++ b/nsis/lang/simpchinese.nsi
@@ -245,14 +245,14 @@ LangString str_msg_compat_vi       ${LANG_SIMPCHINESE} \
 LangString str_msg_compat_vim      ${LANG_SIMPCHINESE} \
     "Vim original"
 LangString str_msg_compat_defaults ${LANG_SIMPCHINESE} \
-    "Vim with some enhancements"
+    "Vim with some enhancements (load defaults.vim)"
 LangString str_msg_compat_all      ${LANG_SIMPCHINESE} \
-    "Vim with all enhancements (Default)"
+    "Vim with all enhancements (load vimrc_example.vim) (Default)"
 
 LangString str_msg_keymap_title   ${LANG_SIMPCHINESE} \
     " Mappings "
 LangString str_msg_keymap_desc    ${LANG_SIMPCHINESE} \
-    "&Remap a few keys (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
 LangString str_msg_keymap_default ${LANG_SIMPCHINESE} \
     "Do not remap keys (Default)"
 LangString str_msg_keymap_windows ${LANG_SIMPCHINESE} \
diff --git a/nsis/lang/tradchinese.nsi b/nsis/lang/tradchinese.nsi
index 5c3ab7d7f..50e0c4cc3 100644
--- a/nsis/lang/tradchinese.nsi
+++ b/nsis/lang/tradchinese.nsi
@@ -246,14 +246,14 @@ LangString str_msg_compat_vi       ${LANG_TRADCHINESE} \
 LangString str_msg_compat_vim      ${LANG_TRADCHINESE} \
     "Vim original"
 LangString str_msg_compat_defaults ${LANG_TRADCHINESE} \
-    "Vim with some enhancements"
+    "Vim with some enhancements (load defaults.vim)"
 LangString str_msg_compat_all      ${LANG_TRADCHINESE} \
-    "Vim with all enhancements (Default)"
+    "Vim with all enhancements (load vimrc_example.vim) (Default)"
 
 LangString str_msg_keymap_title   ${LANG_TRADCHINESE} \
     " Mappings "
 LangString str_msg_keymap_desc    ${LANG_TRADCHINESE} \
-    "&Remap a few keys (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
 LangString str_msg_keymap_default ${LANG_TRADCHINESE} \
     "Do not remap keys (Default)"
 LangString str_msg_keymap_windows ${LANG_TRADCHINESE} \
-- 
2.17.0

